rasa/nlu/components.py

```diff
@@ -397,6 +398,15 @@ def name(self) -> Text:
 
         return type(self).name
 
+    # The unique name can be used to distinguish components in
+    # a pipeline, e.g. when the pipeline contains multiple
+    # featurizers of the same type.
+    @property
+    def unique_name(self) -> Text:
```

Comment:
does it coincide with `file_name` we pass to persist/load?

Reply:
@Ghostvv  It does, apart from the `component_` prefix. Should I get rid of the `self._file_name` method [here](https://github.com/RasaHQ/rasa/blob/74ac0d94c949dfb9b95118d2c15130eba497b169/rasa/nlu/model.py#L243), make the file name optional and use the `unique_name` by default?

Reply:
let's do it in followup PR